app/scripts/metamask-controller.js

```diff
@@ -926,6 +927,22 @@ export default class MetamaskController extends EventEmitter {
       state: initState.AccountOrderController,
     });
 
+    const multichainBalancesControllerMessenger =
+      this.controllerMessenger.getRestricted({
+        name: 'BalancesController',
+        allowedEvents: [],
+        allowedActions: ['SnapController:handleRequest'],
+      });
+
+    this.multichainBalancesController = new MultichainBalancesController({
+      messenger: multichainBalancesControllerMessenger,
+      state: {},
+      listMultichainAccounts:
```

Comment:
Just to match the other comments:

```suggestion
      // TODO: remove when listMultichainAccounts action is available
      listMultichainAccounts:
```